optaplanner-docs/src/main/docbook/en-US/Chapter-Benchmarking_and_tweaking/Chapter-Benchmarking_and_tweaking.xml

```diff
@@ -779,14 +779,14 @@
     &lt;problemBenchmarks&gt;
       ...
       &lt;problemStatisticType&gt;...&lt;/problemStatisticType&gt;
-      &lt;singleStatisticType&gt;PICKED_MOVE_TYPE_BEST_SCORE_DIFF&lt;/singleStatisticType&gt;
+      &lt;subSingleStatisticType&gt;PICKED_MOVE_TYPE_BEST_SCORE_DIFF&lt;/subSingleStatisticType&gt;
```

Comment:
See comment earlier, this won't change, as the type is configured on a single.

Furthermore (regardless that it makes more sens to configure on single than on subsingle), even if we wanted to change this: we can't because the xml is considered backwards compatible... (If and only if using the element in question does not require you to use classes from the impl package. For example the element `<terminationClass>` isn't guaranteed backwards compatible because no one can use it without also using the impl class Termination.)